scripts/all-tests.sh

```diff
@@ -13,7 +13,40 @@ set -e
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
 export JSHINT_THRESHOLD=7550
-export SAFELINT_THRESHOLD=2700
+export SAFELINT_THRESHOLDS='
```

Comment:
@benpatterson: Did you mean reading this from a file that matches the same format as the output of the original report, so one could just copy/paste?  In that case, it would probably be simpler for someone to more quickly bring down these thresholds.  I think the decision we make around safe-commit will make a difference on how important this enhancement is.